Makefile

```diff
@@ -36,6 +36,8 @@ E2E_SC_PARAMS_SOURCE_IMAGE_URL ?= docker://docker.io/library/busybox:latest
 E2E_SC_IMAGE_TAG ?= busybox:latest
 # end-to-end test fully qualified destination image name
 E2E_SC_PARAMS_DESTINATION_IMAGE_URL ?= docker://$(IMAGE_BASE)/${E2E_SC_IMAGE_TAG}
+# --all argument for skopeo copy command
+E2E_SC_PARAMS_ARGS ?= --all
```

Comment:
@Aneesh-M-Bhat can you add tests for other supported field also to show the working behaviour of ARG